### ISSUE TITLE
There should be a 'forgot password' option on the login page. 

### ISSUE THREAD

**Issue by ghost** [bartekpacia/mentorship-client#81]
As a user it is quiet possible that I forget my login password. If that happens I wont be able to login to my account. 

A good solution would be to add a 'Forgot Password' option on the login screen, clicking on which, the user is directed to change their account password via a confirmation link sent to their registered email address. 

An alternative that's possible is; when the user clicks on 'Forgot Password' an OTP is sent to their mobile number that is linked to the account. The user can now log in with the OTP and then change their password by going into 'settings'->'change password'. 

But, I believe the first suggested implementation would be easier. 

![forgot](https://user-images.githubusercontent.com/44665174/82578287-d693a500-9ba9-11ea-8f43-9e91fe69da22.png)

**Comment by techno-disaster** [comment]
@anitab-org/mentorship-backend-maintainers we don't have a forgot password feature yet, right?

**Comment by bartekpacia** [comment]
Nope. It bothers me too a bit. Fortunately there's some work going on [here](https://github.com/anitab-org/mentorship-backend/issues/551).

**Comment by techno-disaster** [comment]
Thanks Bartek, so we can put this on hold for  now

**Comment by techno-disaster** [comment]
> As a user, in case I forget my login password, there should be an option to change it on the login page.

@MaheKhan please use the issue template which can be found [here](https://github.com/anitab-org/mentorship-flutter/tree/develop/.github/ISSUE_TEMPLATE) and reformat your issue body

**Comment by ghost** [comment]
> As a user, in case I forget my login password, there should be an option to change it on the login page.
> 
> @MaheKhan please use the issue template which can be found here and reformat your issue body

okay. Did that.

**Comment by carrycooldude** [comment]
**Is anyone working on this I want to work on this if no-one is doing this....**

**Comment by battuAshita** [comment]
Hi @Techno-Disaster. Is this issue open? If open, can this be assigned to me?

**Comment by KevalPrajapati** [comment]
> Hi @Techno-Disaster. Is this issue open? If open, can this be assigned to me?

Waiting for backend implementation.

**Comment by battuAshita** [comment]
> > Hi @Techno-Disaster. Is this issue open? If open, can this be assigned to me?
> 
> Waiting for backend implementation.

Ok.
